wrappers/services_test.go

```diff
@@ -82,20 +82,22 @@ func (s *servicesTestSuite) SetUpTest(c *C) {
 }
 
 func (s *servicesTestSuite) TearDownTest(c *C) {
-	dirs.SetRootDir("")
-	s.systemctlRestorer()
-	s.delaysRestorer()
 	if s.agent != nil {
 		// Try sending a message to the session agent before
 		// shutting it down to see if that helps with the hang
 		// in gracefully shutting down the agent.
 		cli := client.New()
-		_, err := cli.SessionInfo(context.Background())
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+		_, err := cli.SessionInfo(ctx)
```

Comment:
that we need this feels odd (and is that for all go versions?), also it wasn't maybe even doing what one would think given the restoring happening before it